### PR TITLE
Feature/7 answers polls

### PR DESCRIPTION
**Pull request by lisset-orozco**
- [x] models and validations

- [x] controller for the answers

- [x] show relations

### CODE REVIEW

File: app/controllers/api/v1/my_answers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    # answers controller
+    class MyAnswersController < ApiV1Controller
+      before_action :authenticate
+
+      def create
+        poll = MyPoll.find(params[:my_poll_id])
+        answer = Answer.find(params[:answer_id])
+        user_poll = UserPoll.custom_find_or_create(poll, @current_user)
+        @my_answer = MyAnswer.custom_update_or_create(user_poll, answer)
+
```
Comment (joalbertg) [inline app/controllers/api/v1/my_answers_controller.rb]: delete line

File: app/models/my_app.rb [hunk 2]
```diff
@@ -11,6 +11,7 @@ def your_token?(token)
   end
 
   def valid_origin?(domain)
-    javascript_origins.split(',').include?(domain)
+    binding.pry
```
Comment (joalbertg) [inline app/models/my_app.rb]: delete binding.pry

**Review by joalbertg — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/my_answers_controller.rb (hunk 1 above)
Comment (lisset-orozco) [reply app/controllers/api/v1/my_answers_controller.rb]: done

File: app/models/my_app.rb (hunk 2 above)
Comment (lisset-orozco) [reply app/models/my_app.rb]: done

**Review by joalbertg — APPROVED** [review]
congratulations!!!

**Pull request merged**